event-adapter/pkg/apiserver/installer/installer_test.go

```diff
@@ -1,79 +0,0 @@
-/*
-Copyright 2017 The Kubernetes Authors.
```

Comment:
Please don't remove this test, fix it to work with your case instead. If this is the one that was failing even in the original repo, let's talk offline about your options to make it work.